terraform/modules/sqs/main.tf

```
@@ -0,0 +1,69 @@
+data "aws_caller_identity" "current" {}
+
+# rework as part of #46
+#tfsec:ignore:aws-iam-no-user-attached-policies 
+resource "aws_iam_user" "sqs_user" {
+  name = "${var.environment}-sqs-queue-${var.queue_name}"
```

is this going to have the env in twice then? maybe just do 
```suggestion
  name = "${var.queue_name}-sqs-queue-user"
```